src/SFA.DAS.EmployerIncentives.Data/IncentiveApplication/IncentiveApplicationDataRepository.cs

```diff
@@ -44,5 +48,52 @@ public async Task Update(IncentiveApplicationModel incentiveApplication)
                 _dbContext.AddRange(model.Apprenticeships);
             }
         }
+
+        public async Task<List<IncentiveApplicationModel>> FindApplicationsWithoutApprenticeshipIncentives()
```

Comment:
Isn't this finding applications without EarningsCalculated?  ApprenticeshipIncentives is a term from another bounded context.